docs/api-reference/composite-layer.md

```diff
@@ -34,6 +34,27 @@ Parameters:
 * `props` (Object) - `Layer` properties.
 
 
+## Properties
+
+Inherits from all [Base Layer](/docs/api-reference/layer.md) properties.
+
+##### `subLayerProps` (Object)
+
+Key is the id of a sublayer and value is an object used to override the props of the sublayer.
```

Comment:
Does this support recursive props (i.e. specify a sublayer of a sublayer?) If so maybe show in example/

I thought there was an RFC on this topic, perhaps not needed?

Mark as experimental for now?

Reply:
It does support recursive props. I'll check on the RFC.

Reply:
Looks like it's consistent with the proposal in https://github.com/uber/deck.gl/blob/master/dev-docs/RFCs/vNext/composite-layer-customization-rfc.md